content/api/inline-seeds.apib

```diff
@@ -0,0 +1,511 @@
+
+    + Body
+
+        ```json
+        {
+          "errors": {
+            "message": "Forbidden."
+          }
+        }
+        ```
+
+### List Active Campaigns [GET /inline-seeds/status]
+
+Returns read-only list of seed-eligible campaigns that have recent activity.
+Intended use case is informational - checking status via API or UI.
+Set the X-MSYS-SUBACCOUNT header to get campaigns for a specific subaccount
+(set the header to 0 to get campaigns for the primary account).
+Do not set the X-MSYS-SUBACCOUNT header to get campaigns for the primary
+account and all subaccounts.
+
++ Request - Get campaigns for primary account and subaccounts
+
+    + Headers
+
+        Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+
++ Response 200 (application/json)
+
+    + Body
+
+        ```json
+        {
+          "results": [
+            {
+              "campaign": "my_campaign",
+              "sending_domain": "foo.example.com",
+              "count": 1256,
+              "threshold": 1256,
+              "seed_start": 1620078946682,
+              "duration": 30
+            },
+            {
+              "campaign": "second_campaign",
+              "sending_domain": "bar.example.com",
+              "count": 1337,
+              "threshold": 5000,
+              "seed_start": null,
+              "duration": 30,
+              "subaccount_id": 123
+            }
+          ]
+        }
+        ```
+
++ Request - Get campaigns for subaccount
+
+    + Headers
+
+        Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+        X-MSYS-SUBACCOUNT: 123
+
++ Response 200 (application/json)
+
+    + Body
+
+        ```json
+        {
+          "results": [
+            {
+              "campaign": "second_campaign",
+              "sending_domain": "bar.example.com",
+              "count": 1337,
+              "threshold": 5000,
+              "seed_start": null,
+              "duration": 30,
+              "subaccount_id": 123
+            }
+          ]
+        }
+        ```
+
++ Request - Get campaigns for primary account
+
+    + Headers
+
+        Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+        X-MSYS-SUBACCOUNT: 0
+
++ Response 200 (application/json)
+
+    + Body
+
+        ```json
+        {
+          "results": [
+            {
+              "campaign": "my_campaign",
+              "sending_domain": "foo.example.com",
+              "count": 1256,
+              "threshold": 1256,
+              "seed_start": 1620078946682,
+              "duration": 30
+            }
+          ]
+        }
+        ```
+
+### Check Campaign Status [GET /inline-seeds/status?id={campaign_id}&sending_domain={sending_domain}]
+
+Returns read-only campaign metadata.
+Auth context (from API key) provides customer and subaccount ids.
+
++ Parameters
+    + id: `my-campaign` (string, required)
+    + sending_domain: `foo.example.com` (string, required)
+
++ Request
```

Comment:
Same question as above: in the sample request, is it possible to use a sample campaign ID rather than {campaign_id}?